pkg/tests/BUILD

```diff
@@ -12,9 +12,35 @@ genrule(
     cmd = "for i in $(OUTS); do echo 1 >$$i; done",
 )
 
+pkg_tar(
+    name = "test-tar-package-dir",
```

Comment:
Please use _ instead of - as a separator to be consistent with the other code.

Reply:
@aiuto good point! addressed in https://github.com/bazelbuild/rules_pkg/pull/57/commits/fa82eea821e9cadd65cc994215e8c7fdb08c5284